persist-cli/src/main/java/io/ballerina/persist/cmd/Migrate.java

```diff
@@ -312,24 +317,56 @@ private static void migrateWithTimestamp(File migrationsDir, String migrationNam
             Module previousModel = BalProjectUtils.getEntities(previousModelPath);
             Module currentModel = BalProjectUtils.getEntities(currentModelPath);
 
-            queries = findDifferences(previousModel, currentModel);
+            queries = findDifferences(previousModel, currentModel, dataStore);
 
             // Write queries to file
             if (!queries.isEmpty()) {
-                String filePath = Paths.get(newMigrationPath.toString(), "script.sql").toString();
+                String filePath = Paths.get(newMigrationPath.toString(), "script" + 
+                    checkDataStore(dataStore)).toString();
                 try (FileOutputStream fStream = new FileOutputStream(filePath);
                         OutputStreamWriter oStream = new OutputStreamWriter(fStream, StandardCharsets.UTF_8);
                         BufferedWriter writer = new BufferedWriter(oStream)) {
-                    writer.write("-- AUTO-GENERATED FILE." + System.lineSeparator() +
+                    switch(dataStore) {
+                        case "mysql":
+                            writer.write("-- AUTO-GENERATED FILE." + System.lineSeparator() +
                             "-- This file is an auto-generated file by Ballerina " +
                             "persistence layer for the migrate command." + System.lineSeparator() +
                             "-- Please verify the generated scripts and " +
                             "execute them against the target DB server." +
                             System.lineSeparator() + System.lineSeparator());
+                            break;
+
+                        case "googlesheets":
+                            writer.write("// AUTO-GENERATED FILE." + System.lineSeparator() +
+                            "// This file is an auto-generated file by Ballerina " +
+                            "persistence layer for the migrate command." + System.lineSeparator() +
+                            "// Please verify the generated scripts and " +
+                            "execute them against the target DB server." +
+                            System.lineSeparator() + System.lineSeparator() +
+                            "function migrateSheets() {" + System.lineSeparator() +
+                            "   var activeSpreadsheet = SpreadsheetApp.getActiveSpreadsheet();" +
+                            "   var yourNewSheet = \"\"");
```

Comment:
why code is added here?

Reply:
I made them as they are recurring variables in the script, so i initialized them both before adding any queries